go/vt/vtgate/vcursor_impl.go

```diff
@@ -477,6 +490,9 @@ func (vc *vcursorImpl) TabletType() topodatapb.TabletType {
 
 // SubmitOnlineDDL implements the VCursor interface
 func (vc *vcursorImpl) SubmitOnlineDDL(onlineDDl *schema.OnlineDDL) error {
+	if vc.topoServer == nil {
+		return vterrors.New(vtrpcpb.Code_INTERNAL, "Unable to apply DDL, missing toposerver in vcursor")
```

Comment:
one nit - this error might be too cryptic for an end user. I wonder if we should mention that probably they have a keyspace filter and that is incompatible with online DDLs

Reply:
Fair enough -- I referenced filtered keyspaces in the error but since it's a bit indirect i'm going to leave the general commentary around why the failure is happening as well.